HTML/html_completions.py

```diff
@@ -190,6 +191,9 @@ class HtmlTagCompletions(sublime_plugin.EventListener):
     Provide tag completions for HTML
     It matches just after typing the first letter of a tag name
     """
+
+    KIND_ENTITY = (sublime.KIND_ID_MARKUP, 'e', 'Entity')
+
```

Comment:
Maybe add a `KIND_SNIPPET = (sublime.KIND_ID_SNIPPET, 'e', 'Entity')` here.

May also be named `KIND_ENTITY_SNIPPET`?

Reply:
I think I did these slightly differently than you did. Any objections?